### ISSUE TITLE
Add command line option to import environment variables

### ISSUE THREAD

**Issue by bergos** [zazuko/barnard59#46]
Most of the time pipeline variables are populated from environment variables. There should be a short alternative way to assign env variables to pipeline variables with the same name. It should be possible to do this by variable name or for all variables.

At the moment it would look like this:

```bash
barnard59 --variable ENDPOINT=$ENDPOINT,USER=$USER,PASSWORD=$PASSWORD
```

A shorter version could skip the value argument and use the env variable with the same name for that case:

```bash
barnard59 --variable ENDPOINT,USER,PASSWORD
```

An even short version could just import all env variables:

```bash
barnard59 --variable-all
```

**Comment by tpluscode** [comment]
Is this a more limited alternative to #42?

The environment is anyway available to the running process. I think it's more useful to have the ENV <-> variable mapping inside the pipeline definition

The proposed CLI option also does not cater for programmatic execution, such as how we run the pipeline in cube creator

**Comment by bergos** [comment]
#42 requires to add runtime specific features to the pipeline definition vocab. That's what I would like to avoid. The runtime should be responsible to fill the variable map in a way that makes sense for the environment. That step will be different for the browser, cli, or a more customized node use case. Adding this to the pipeline core would increase the complexity of the code and may require polyfill workarounds for environments which don't have env variables.

**Comment by bergos** [comment]
fixed in #51
